src/main/scala/com/fulcrumgenomics/fastq/FastqSource.scala

```diff
@@ -59,14 +59,14 @@ object FastqSource {
     * @param sources a Seq of one or more FastqSource objects
     * @return an Iterator that returns a Seq of FastqRecord, one per source
     */
-  def zipped(sources: Seq[FastqSource]): Iterator[Seq[FastqRecord]] = new Iterator[Seq[FastqRecord]] {
+  def zipped(sources: Seq[Iterator[FastqRecord]]): Iterator[Seq[FastqRecord]] = new Iterator[Seq[FastqRecord]] {
     require(sources.nonEmpty, "No sources provided")
 
     def hasNext(): Boolean = sources.exists(_.hasNext)
 
     def next(): Seq[FastqRecord] = {
       if (!this.hasNext) throw new NoSuchElementException("Calling next() when hasNext() is false.")
-      require(sources.forall(_.hasNext) == sources.head.hasNext, "Sources are out of sync.")
+      require(sources.forall(_.hasNext) == sources.head.hasNext, s"Sources are out of sync: ${sources.map(_.hasNext).mkString(", ")}")
```

Comment:
I tend to think messages in require() statements should make sense without having to look at code - e.g. a user could see this if they feed in a truncated fastq file.  If you want to add more detail you could figure out which one is out of sync and say something like `first and nth fastq file are out of sync`.